src/main_dpdk.cpp

```diff
@@ -909,6 +921,23 @@ COLD_FUNC static int parse_options(int argc, char *argv[], bool first_time ) {
             case OPT_SLEEPY_SCHEDULER:
                 CGlobalInfo::m_options.m_is_sleepy_scheduler = true;
                 break;
+            case OPT_LATENCY_MEASUREMENT_METHOD:
+                sscanf(args.OptionArg(),"%d", &tmp_data);
```

Comment:
Please, use spaces between arguments (also you can fix it in other options)

Reply:
I tried to follow the convention.  But maybe tried to hard.  I guess I can fix my code.